src/plugins/github/fetchGithubRepo.js

```diff
@@ -142,7 +142,14 @@ type GithubResponseError =
 // Fetch against the GitHub API with the provided options, returning a
 // promise that either resolves to the GraphQL result data or rejects
 // to a `GithubResponseError`.
-function tryGithubFetch(fetch, fetchOptions): Promise<any> {
+function tryGithubFetch(postBody: string, token: string): Promise<any> {
```

Comment:
may as well use the validated GithubToken type: https://github.com/sourcecred/sourcecred/blob/0296bf38e2e7ece574135a58e0f209dd16562242/src/plugins/github/token.js#L3

Reply:
Ah, I’d forgotten about it—thanks!